cypress/e2e/2-advanced-examples/registrantionStudentForm_TC.cy.js

```diff
@@ -0,0 +1,179 @@
+/// <reference types='cypress'/>
+
+import { Registpage, confirmform } from "../pages/Registpage";
+
+
+describe('demoQA - Regist Form - Sucessfully', () => {
+  beforeEach(() => {
+    cy.fixture("account").as("user");
+    cy.visit('https://demoqa.com/automation-practice-form');
+    
+  })
+  
+  // HAPPY CASE
+  it ('User filling all valid infor', () => {
+      cy.get("@user").then((user) => {
+        Registpage
+        .typeFirstname(user.firstName)
+        .typeLastname(user.lastName)
+        .typeEmail(user.email)
+        .selectGender(user.gender)
+        .typeUserMobile(user.phoneNumber)
+        .chooseDOB(user.month, user.year, user.date)
+        .chooseSubjects(user.subjects)
+        .selectHobbies(user.hobby)
+        .choosePiture(user.file1)
+        .typeAddress(user.address)
+        .typeState(user.state)
+        .typeCity(user.city)
+        .clickSubmit()
+
+        confirmform
+        //.isNameCorrect(user.firstName+ " " + user.lastName)
+        .isMailCorrect(user.email)
+        .isGenderCorrect(user.gender)
+        .isMobileCorrect(user.phoneNumber)
+        .isDOBCorrect(user.date, user.month, user.year)
+        .isSubjectsCorrect(user.subjects)
+        .isHobbiesCorrect(user.hobby)
+        //.isPictureCorrect(user.file1)
+        .isAddressCorrect(user.address)
+        .isState_and_City_Correct(user.state+ " " + user.city)
+      });
+      cy.get('#closeLargeModal').click({force:true})
+    });
+
+    it ('User filling requirement information only', () => {
+      cy.get("@user").then((user) => {
+        Registpage
+        .typeFirstname(user.firstName)
+        .typeLastname(user.lastName)
+        .selectGender(user.gender)
+        .typeUserMobile(user.phoneNumber)
+        .clickSubmit()
+
+        confirmform
+        //.isNameCorrect(user.firstName+ " " +user.lastName)
+        .isGenderCorrect(user.gender)
+        .isMobileCorrect(user.phoneNumber)
+      });
+      cy.get('#closeLargeModal').click({force:true})
+    });
+
+  });
+  
+// UNHAPPY CASE  
+  describe('demoQA - Regist Form - Unsucessfully', () => {
```

Comment:
Happy cases and unhappy cases are still in the same test suite, wrap them in one `describe()`